metricbeat/module/elasticsearch/_meta/fields.yml

```diff
@@ -613,7 +613,26 @@
                   type: alias
                 - name: rejected
                   path: elasticsearch.node.stats.thread_pool.write.rejected.count
-                  type: alias
+                  type: alias    
+        - name: indexing_pressure
```

Comment:
This entire group is not required

Reply:
@sayden Is my understanding correct that the group only has to exist in `node_stats/_meta/fields.yml`?